pkg/api/views.go

```diff
@@ -21,17 +21,29 @@ func (x PartsHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	}
 
 	type tableRow struct {
-		Project      string
-		PartName     string
-		PartNumber   uint8
-		SheetMusic   string
-		ClickTrack   string
-		BackingTrack string
+		Project      string `json:"project"`
+		PartName     string `json:"part_name"`
+		PartNumber   uint8  `json:"part_number"`
+		SheetMusic   string `json:"sheet_music"`
+		ClickTrack   string `json:"click_track"`
+		BackingTrack string `json:"backing_track"`
 	}
 
-	allParts := x.Parts.List()
-	rows := make([]tableRow, 0, len(allParts))
-	for _, part := range allParts {
+	parts := x.Parts.List()
+	want := len(parts)
+	for i := 0; i < want; i++ {
+		if parts[i].Validate() == nil &&
+			projects.GetName(parts[i].Project).Archived == false &&
+			projects.GetName(parts[i].Project).Released == true {
```

Comment:
```suggestion
			!projects.GetName(parts[i].Project).Archived &&
			projects.GetName(parts[i].Project).Released {
```
I'm 50% sure this passes `gofmt`.

Reply:
Personally, I find that less readable.

Reply:
I guess it (probably?) gets optimized out anyway.